Dockerfile

```diff
@@ -1,10 +1,14 @@
 FROM docker.io/python:3.9-slim-buster
 
-WORKDIR		/root/app
+WORKDIR /root/app
 
-COPY	. .
+# Caching the dependencies
+COPY requirements.txt ./
```

Comment:
`COPY . .` does this just put the `RUN` after the `COPY . .`

Reply:
What this does is just coping ```requirements.txt```, then it's installing the dependencies.
since the deps don't change that often, this will get cached.

But when we do ```COPY . .``` the whole project will get copied, and if there was a change (and there is always a change since we will not try to rebuild the project if we don't change anything), there for the "cache" will never match, and there is basically no cache

Reply:
cool